packages/wonder-blocks-timing/docs.md

```diff
@@ -1,11 +1,398 @@
+            </Unmounter>
+            <View>
+                <View id={id}></View>
+            </View>
+        </View>
+    )}
+</IDProvider>
+```
+
+But if we use `withActionScheduler` and the `interval` method, everything is
+just fine. Unmount the component, and the logging stops.
+
+```jsx
+const {withActionScheduler} = require("@khanacademy/wonder-blocks-timing");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {IDProvider, View} = require("@khanacademy/wonder-blocks-core");
+
+
+class Unmounter extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            mountKids: true,
+        };
+    }
+
+    maybeRenderKids() {
+        if (this.state.mountKids) {
+            return (
+                <React.Fragment>
+                    <Button onClick={() => this.onClick()}>Unmount</Button>
+                    {this.props.children}
+                </React.Fragment>
+            );
+        } else {
+            return "Children unmounted";
+        }
+    }
+
+    onClick() {
+        this.setState({mountKids: false});
+    }
+
+    render() {
+        return (
+            <View>
+                {this.maybeRenderKids()}
+            </View>
+        );
+    }
+}
+
+class MyGoodComponent extends React.Component {
+    componentDidMount() {
+        const {targetId, schedule} = this.props;
+        let counter = 0;
+        const domElement = document.getElementById(targetId);
+        schedule.interval(() => {
+            domElement.innerText = "Naughty interval logged: " + counter++;
+        }, 200);
+    }
+
+    render() {
+        return <View>GoodComponent here</View>;
+    }
+}
+
+const MyGoodComponentWithScheduler = withActionScheduler(MyGoodComponent);
+
+<IDProvider>
+    {id => (
+        <View>
+            <Unmounter>
+                <MyGoodComponentWithScheduler targetId={id} />
+            </Unmounter>
+            <View>
+                <View id={id}></View>
+            </View>
+        </View>
+    )}
+</IDProvider>
+```
+
+<!-- Styleguidist doesn't support the extended syntax for custom header IDs.
+     If that ever changes, this should be:
+         `### API Overiview {#timing-api-overview}`
+-->
+<h3 id="timing-api-overview">API Overview</h3>
+
+#### IScheduleActions
+
+The `IScheduleActions` interface provides 4 (four) different functions:
+
+* [`timeout`](#timeout)
+* [`interval`](#interval)
+* [`animationFrame`](#animationframe)
+* [`clearAll`](#clearall)
+
+##### timeout
+
+```js static
+    timeout(
+        action: () => mixed,
+        period: number,
+        autoSchedule?: boolean,
+        resolveOnClear?: boolean,
+    ): ITimeout;
+```
+
+The `timeout` function replaces the `setTimeout` and `clearTimeout` functions
+in the standard timer API.
+
+Unless indicated otherwise, all timeouts scheduled this way will be cleared on component unmount.
+
+| | Flow&nbsp;Type | Default | Description |
```

Comment:
cool usage of md tables! curious to know if you use a tool to write them down or did it manually.

Reply:
I did it manually.